br/pkg/lightning/checkpoints/checkpoints_sql_test.go

```diff
@@ -431,23 +431,23 @@ func TestDestroyAllErrorCheckpoints_SQL(t *testing.T) {
 
 	s.mock.ExpectBegin()
 	s.mock.
-		ExpectQuery("SELECT (?s:.+)'all' = \\?").
```

Comment:
the old case is checking there's a `'all'`, so I think we don't need to check the SQL pattern. Only checking the corresponding argument is `'all'` is enough.

Reply:
since `WHERE %[2]s = ?` has been changed to `WHERE ? = ?`, now it will not contain `'all'`, I have changed to check `? = ?`

Reply:
I mean the argument at line 435. We should check it's `'all'` rather than any value.

Reply:
`selectQuery` in `DestroyErrorCheckpoint` has been changed, where `WHERE %[2]s = ?` has been changed to `WHERE ? = ?`, previously `'all'` was put into `selectQuery` by `%[2]s`, so now it will not contain `'all'`

Reply:
The query argument must contain `'all'`, otherwise we mistakenly changed the logic 🤔